### ISSUE TITLE
Supporting HAIP as a mandatory?

### ISSUE THREAD

**Issue by endimion** [EWC-consortium/eudi-wallet-rfcs#77]
This issue is meant to open a discussion with respect to the use of HAIP in EWC. 

In [RFC001 v2](https://github.com/EWC-consortium/eudi-wallet-rfcs/blob/main/ewc-rfc001-issue-verifiable-credential.md) we mention supportitng HAIP. 

The [ARF v1.4](https://eu-digital-identity-wallet.github.io/eudi-doc-architecture-and-reference-framework/1.4.0/arf/) includes HAIP  but only as a reference , we should make a decision as to if HAIP is mandator or optional  (with respect to EWC piloting). 

This obviously affects various parts of the RFC001 such requirement for Pushed Authorisation Request  and so on...  

Let us try to get a concensous ....

**Comment by ntsbs** [comment]
The intention of HAIP was also to restrict the options and therefore increase the likeliness of interop. I am in favor of making it mandatory. Any issues to be aware of here?

**Comment by andreasabr** [comment]
Yes, I fully agree but we have to keep in mind that there is a misalignment between HAIP and OIDC4VCI which needs to be considered.

**Comment by endimion** [comment]
Hey all, @andreasabr would you be kind enough to point to these issues so we can discuss (to be honest I was not aware)
